src/bootstrap.js

```diff
@@ -129,10 +148,13 @@ suite.on("complete", event => {
 });
 
 suite.on("error", event => {
+  window.automated.completed = true;
   const benchmark = event.target;
   const error = benchmark.error;
   const name = benchmark.name;
-  document.body.innerHTML = `<h1>ERROR</h1><p>Encountered errors during execution of ${name} test. Refusing to run a partial benchmark suite.</p><pre>${error.stack}</pre>`;
```

Comment:
Also undo these unrelated changes.

Reply:
Similar to the above, I think this is just prettier doing its thing. In that case, we should keep the changes.